packages/nextjs/components/impact-vector/ImpactVectorGraph.tsx

```diff
@@ -6,6 +6,7 @@ import {
   Area,
   AreaChart,
   CartesianGrid,
+  Legend,
```

Comment:
Can you remove this? I merged in main branch and removed the reference to legend below because I don't think we will need it. I failed to remove this import though and the pipeline is failing as a result.